README.md

```diff
@@ -1,17 +1,10 @@
 # Baby Steps with Rust - UDP Datagram
 
-Hey there! I'm Tim. I primarily work as a Ruby on Rails developer, but I've been excited by Rust for a long time. And I have dabbled. Oh, how I have dabbled. But recently I've felt like it was time to write some more substantial Rust code, so I came up with some babysteps...
-
-Any this is the first babystep! .
-
 One thing that attracts me to Rust is that it is a systems language that you can also comfortably write web apps in. So, I wanted to start with something systems-y, but very small. Something with bytes! This led me to chose a very simple networking protocol: UDP.
 
 UDP is User Datagram Protocol, and was created as part of the Internet Protocol Suite.
 
-Breaking down its name further we learn that:
```

Comment:
This reads too close to the cliche of using dictionary definitions to start off  videos and articles - I don't think this specific spelling-out adds much.